group2_assignment2_solution.ipynb

```diff
@@ -495,7 +507,10 @@
     "you have crawled for each keyword, save them into a file named w.json where w is the keyword.\n",
     "'''\n",
     "# saving the tweets in three json files, one for each keyword\n",
-    "#INSERT YOUR CODE HERE"
+    "#INSERT YOUR CODE HERE\n",
+    "save_to_json(k1_tweets, 'group2_computer.json')\n",
+    "save_to_json(k2_tweets, 'group2_animal.json')\n",
+    "save_to_json(k3_tweets, 'group2_movie.json')"
```

Comment:
does something like
for index, keyword in enumerate(keywords):
    save_to_json(keyword{i + 1}_tweets, "group2_{keyword}.json")

work?